src/components/input-field/input-field.scss

```diff
@@ -45,25 +45,33 @@
     }
 }
 
+@include textfield.core-styles;
```

Comment:
I don't know on which line I should give this comment but the floating label that should be above in case of textarea doesn't look good. 

<img width="962" alt="Screenshot 2021-07-15 at 11 04 01" src="https://user-images.githubusercontent.com/22968497/125762178-05c63955-8e53-4f8c-9aba-c59ccdf128b8.png">

Reply:
Oh yeah, good catch! I've seen this but had forgotten about it.

(Don't worry about what line to write a comment on. As long as it's on _some_ line, so answers can be threaded, I'm happy ☺️)

Reply:
I'll mark this as resolved, not because it's fixed, but because I've added it to the list in the main description.